Fix Pipe ID printf to be strict type-safe compatible

**Is your feature request related to a problem? Please describe.**
The `CFE_SB_PipeId_t` type is not a simple/plain integer if strict types are enabled.  This trips up a printf statement in TO_LAB here where it tries to print the `TO_LAB_Global.Tlm_pipe` value:

```
/home/joe/code/cfecfs/github/apps/to_lab/fsw/src/to_lab_app.c:507:77: error: format ‘%d’ expects argument of type ‘int’, but argument 6 has type ‘CFE_SB_PipeId_t’ [-Werror=format=]
  507 |                           "L%d TO Can't Unsubscribe to Stream 0x%x on pipe %d, status %i", __LINE__,
      |                                                                            ~^
      |                                                                             |
      |                                                                             int
  508 |                           (unsigned int)CFE_SB_MsgIdToValue(pCmd->Stream), TO_LAB_Global.Tlm_pipe, (int)status);
      |                                                                            ~~~~~~~~~~~~~~~~~~~~~~
      |                                                                                         |
      |                                                                                         CFE_SB_PipeId_t
```

**Describe the solution you'd like**
Use the `CFE_RESOURCEID_TO_ULONG()` macro to get the simple value, and use `%lu` conversion specifier in printf.

**Requester Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/fsw/src/to_lab_app.c
+++ b/fsw/src/to_lab_app.c
@@ -504,8 +504,8 @@ int32 TO_LAB_RemovePacket(const TO_LAB_RemovePacketCmd_t *data)
     status = CFE_SB_Unsubscribe(pCmd->Stream, TO_LAB_Global.Tlm_pipe);
     if (status != CFE_SUCCESS)
         CFE_EVS_SendEvent(TO_REMOVEPKT_ERR_EID, CFE_EVS_EventType_ERROR,
-                          "L%d TO Can't Unsubscribe to Stream 0x%x on pipe %d, status %i", __LINE__,
-                          (unsigned int)CFE_SB_MsgIdToValue(pCmd->Stream), TO_LAB_Global.Tlm_pipe, (int)status);
+                          "L%d TO Can't Unsubscribe to Stream 0x%x, status %i", __LINE__,
+                          (unsigned int)CFE_SB_MsgIdToValue(pCmd->Stream), (int)status);
     else
         CFE_EVS_SendEvent(TO_REMOVEPKT_INF_EID, CFE_EVS_EventType_INFORMATION, "L%d TO RemovePkt 0x%x", __LINE__,
                           (unsigned int)CFE_SB_MsgIdToValue(pCmd->Stream));

